commitlintplugins.test.ts

```diff
@@ -423,6 +423,27 @@ test('subject-lowercase10', () => {
 });
 
 
+test('title-uppercase1', () => {
+    let commitMsgWithoutArea = "remove logs";
+    let titleUpperCase1 = runCommitLintOnMsg(commitMsgWithoutArea);
+    expect(titleUpperCase1.status).not.toBe(0);
+});
+
+
+test('title-uppercase2', () => {
+    let commitMsgWithoutArea = "Remove logs";
+    let titleUpperCase2 = runCommitLintOnMsg(commitMsgWithoutArea);
+    expect(titleUpperCase2.status).toBe(0);
+});
+
+
+test('title-uppercase3', () => {
+    let commitMsgWithoutArea = "testFixtureSetup refactor";
+    let titleUpperCase3 = runCommitLintOnMsg(commitMsgWithoutArea);
+    expect(titleUpperCase3.status).toBe(0);
+});
```

Comment:
let's add another test: title-uppercase4 which has this title "lowercase: area is lowercase" whose status should be 0.

Reply:
Done

Reply:
Done